docker/Dockerfile.notifications-connector-email.jvm

```diff
@@ -0,0 +1,40 @@
+####
```

Comment:
Things changed several times in the Dockerfiles recently. Could you please update that one to match the latest version from `master`? You can use any file from a connector as an example.

Reply:
Yes, I've updated the file to match what we have in other connectors!